### PR TITLE
add Ghost chart

### PR DESCRIPTION
**Pull request by prydonius**
Ghost needs to know the external host of the application at deploy-time so it can configure the application URLs correctly. I've been battling with different ways to solve this issue, and I've opted for a similar approach to @gtaylor's approach to requiring EULA in Minecraft. Essentially, if you don't provide the release with an external host, it will skip the deployment and tell you how to obtain the external IP from the service created and upgrade the deployment.

I feel this is a good approach because it makes it clear that the deployment is incomplete, and forces users to configure the correct host before using the application in a broken state.

That said, I'm open to hearing other ideas on solving this problem. One other idea I had, and implemented in a separate workspace, was to use a post-install hook to launch a pod that uses the Kubernetes API to automatically configure the correct host. The issue with this is that it relies on automatic load balancer provisioning, is not obvious to the user, and is difficult to maintain.

cc @viglesiasce @migmartri @sameersbn @michelleN @lachie83

### CODE REVIEW

**Comment by lachie83** [comment]
Thanks for the contribution @prydonius Will review!

File: stable/ghost/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{{- if empty (include "host" .) -}}
+###############################################################################
+### ERROR: You did not provide an external host in your 'helm install' call ###
+###############################################################################
+
+This deployment will be incomplete until you configure Ghost with a resolvable
+host. To configure Ghost with the URL of your service:
+
+1. Get the Ghost URL by running:
```
Comment (viglesiasce) [inline stable/ghost/templates/NOTES.txt:9]: Can you note somewhere that in order to login you need to go to /admin?

**Review by viglesiasce — CHANGES_REQUESTED** [review]

File: stable/ghost/templates/NOTES.txt (hunk 1 above)
Comment (prydonius) [reply stable/ghost/templates/NOTES.txt:9]: Added links to `/ghost` as well as `/` below :)

File: stable/ghost/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  {{- if contains "NodePort" .Values.serviceType }}
+
+  export APP_PORT=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.spec.ports[0].nodePort}")
+  export APP_HOST=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+
+  {{- else if contains "LoadBalancer" .Values.serviceType }}
+
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc -w {{ template "fullname" . }}'
```
Comment (sameersbn) [inline stable/ghost/templates/NOTES.txt:19]: I think you should add `--namespace {{ .Release.Namespace }}` to the command?

Comment (viglesiasce) [reply stable/ghost/templates/NOTES.txt:19]: yep

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**